tsconfig.json

```diff
@@ -3,7 +3,7 @@
     "incremental": true,
     "composite": true,
     "target": "es6",
-    "module": "es6",
+    "module": "es2020",
```

Comment:
Changed to ES2020 to support dynamic imports (aka lazy loading).

_Changing `module` affects what import/export syntax Typescript will let us use. Webpack still transpiles it to something with wider browser support._